Makefile.work

```diff
@@ -346,6 +358,14 @@ showtag:
 	@echo $(SLAVE_IMAGE):$(SLAVE_TAG)
 	@echo $(SLAVE_BASE_IMAGE):$(SLAVE_BASE_TAG)
 
+initpde:
```

Comment:
this does not seem necessary, why not checkout all the source code.

Reply:
This is for generating light weight SONIC for co-working with ODM on the HWQ.
We will run make initpde to generate the SONIC PDE source package.

Reply:
The git operations should be out of the scope of Makefile. Every developer should recursively fetch the full source tree, if you want to build something in small scope, you don't need to limit the code checkout process.